src/main/resources/META-INF/plugin.xml

```diff
@@ -16,8 +16,8 @@
   <p><b>0.0.1</b></p>
   <p>Initial release</p>
   <p>Allow create, edit and start actions for pipelines, tasks and resources</p>
-  <p><b>WARNING</b>: this plugin has been tested and validated against Tekton Pipelines Alpha1.
-  You may have some issues against Tekton Pipelines Beta1</b>
+  <p><b>WARNING</b>: this plugin has been tested and validated against Tekton Pipelines Beta1.
+  You may have some issues against Tekton Pipelines Alpha1</b>
```

Comment:
We should be more strict for Alpha1

Reply:
What do you mean?

Reply:
We should claim this plugin requires Pipeline Beta1 as a minimum.

Reply:
Ah ok. Yeah it makes sense. I guess i'm going to revert this change and open a new issue then. In the change-notes we should also add that you need to install triggers if you want to use them. I see all of this together when we're going to add the new plugin-version in the change-notes. I also discovered a user can install a specific version of a plugin so it makes sense to leave this warning message as it is for the first version and add a new one with the next release

Reply:
Opened https://github.com/redhat-developer/intellij-tekton/issues/118